logstage/logstage-core/.jvm/src/main/scala/izumi/logstage/adapter/jul/LogstageJulLogger.scala

```diff
@@ -42,25 +42,27 @@ class LogstageJulLogger(router: LogRouter) extends java.util.logging.Handler wit
 
   @inline private[this] def mkEntry(record: LogRecord): Log.Entry = {
 
-    val id = Log.LoggerId(record.getLoggerName)
+    val loggerName = if (record.getLoggerName == null) "unknown" else record.getLoggerName
+    val id = Log.LoggerId(loggerName)
 
     val thread = Thread.currentThread()
 
     val ctx = Log.StaticExtendedContext(id, SourceFilePosition.unknown)
     val threadData = Log.ThreadData(thread.getName, thread.getId)
 
-    val allParams = if (record.getThrown == null) {
-      record.getParameters.toSeq
-    } else {
-      record.getParameters.toSeq ++ Seq(record.getThrown)
-    }
+    val params = if (record.getParameters == null) Seq.empty else record.getParameters.toSeq
```

Comment:
The same, `Option().getOrElse()`

Reply:
@pshirshov I've thought about this but decided that it's a better solution for performance. If you insist I can change null checks to options.